src/QsCompiler/DocumentationParser/DocComment.cs

```diff
@@ -317,8 +321,36 @@ void ParseMapSection(IEnumerable<Block> blocks, Dictionary<string, string> accum
                     }
                 }
 
+                if (deprecated && !deprecationProcessed)
+                {
+                    var deprecationText = DeprecatedWarning + "\r" + deprecation;
+                    this.Summary += "\r" + deprecationText;
+                    this.ShortSummary = deprecationText;
+                    summarySection.AddRange(DeprecatedSection);
+                    if (!string.IsNullOrEmpty(deprecation))
+                    {
+                        summarySection.AddRange(Markdown.Parse(deprecation));
+                    }
+                }
+
                 this.Documentation = ToMarkdown(summarySection.Concat(descriptionSection));
             }
+            else
```

Comment:
Am I missing something or is this else case generating a deprecated warning if there are no doc comments on a declaration even if that declaration has not been marked as deprecated?

Reply:
No, only if there are no doc comments and there was a Deprecated attribute. This has changed a bit, though.